### ISSUE TITLE
[Banco de Séries] Validate the application accessibility failed

### ISSUE THREAD

**Issue by diegohdb** [diegohdb/QA-ManualTesting#3]
#### Precondition
1- Have a PC connected to network
#### Steps to Reproduce:

| Step | Action | Expected | Status |
| -------- | -------- | -------- | -------- |
| 1| 1- Launch <a href="https://bancodeseries.com.br">https://bancodeseries.com.br</a>| 1- The website is loaded fine| Pending |
| 2| 2- Copy the home address and paste into <a href="https://achecker.ca/checker/">https://achecker.ca/checker/</a> and click ‘Check it’  | 2- The check result will not detect any non conformity| Pending |

**Comment by diegohdb** [comment]
The results returned from Accessibility Checker is following:

[achecker_2020-10-14_19-41-16.pdf](https://bitmodern-testquality-server-storage.s3.us-west-2.amazonaws.com/attachment_RunResult_15526.pdf)

![](https://bitmodern-testquality-server-storage.s3.us-west-2.amazonaws.com/attachment_RunResult_15525.png)

**Comment by diegohdb** [comment]
https://diegohdb.testquality.com defect [D3](https://diegohdb.testquality.com/project/11267/result/2134124/defect/7253). Do not remove or edit.
